### PR TITLE
error fix #6a5ba65

### PR DESCRIPTION
**Pull request by subnwa**
Although the OPENAPI key was previously registered, it was getting an error and I could not find the solution. Now, with a small change, the problem has completely disappeared and now it works flawlessly.

### CODE REVIEW

File: gpt_engineer/main.py [hunk 1]
```diff
@@ -51,13 +51,12 @@ def chat(
...
     )
 
     for step in STEPS[steps_config]:
         messages = step(ai, dbs)
         dbs.logs[step.__name__] = json.dumps(messages)
 
-
 if __name__ == "__main__":
-    app()
+    app.run()
```
Comment (huntersinclairtiv) [inline gpt_engineer/main.py:62]: I get an error on this -  says typer run() does not exist

Comment (subnwa) [reply gpt_engineer/main.py:62]: Yeah I guess that's why it's giving an error. I need to find another solution for OPENAPI error

Comment (patillacode) [reply gpt_engineer/main.py:62]: Yeah, I don't think this works for me either.

Also, I think your branch is not up to date with `main` @subnwa

Comment (subnwa) [reply gpt_engineer/main.py:62]: I realized that there is an error in my own system, I will review it again.

**Comment by AntonOsika** [comment]
I don't understand.

Feel free to open with a clearer description of what you are solving and how it solves it

**Pull request closed without merging**